tests/test_helper.py

```diff
@@ -1,17 +1,25 @@
 from pathlib import Path
-from normalizer import helper
+from normalizer import helper as normalizer_helper
+from generator import helper as generator_helper
+import tempfile
 
 cur_dir = Path(__file__).parent
 
 
 def test_choose_jina_version(mocker):
     mocker.patch('normalizer.helper.get_jina_latest_version', return_value='2.0.0rc10')
 
-    assert helper.choose_jina_version('2.0.0rc11') == '2.0.0rc10'
-    assert helper.choose_jina_version('2.0.0rc9') == '2.0.0rc9'
-    assert helper.choose_jina_version('master') == 'master'
+    assert normalizer_helper.choose_jina_version('2.0.0rc11') == '2.0.0rc10'
+    assert normalizer_helper.choose_jina_version('2.0.0rc9') == '2.0.0rc9'
+    assert normalizer_helper.choose_jina_version('master') == 'master'
 
 
 def test_convert_from_path():
-    assert helper.convert_from_to_path('..deps', base_dir=cur_dir / 'cases/nested_3/executors')
-    assert helper.convert_from_to_path('deps', base_dir= cur_dir / 'cases/nested_3')
\ No newline at end of file
+    assert normalizer_helper.convert_from_to_path('..deps', base_dir=cur_dir / 'cases/nested_3/executors')
+    assert normalizer_helper.convert_from_to_path('deps', base_dir= cur_dir / 'cases/nested_3')
+
+
+def test_to_jcloud_yaml():
+    (fp, temp_file_path) = tempfile.mkstemp()
+    assert generator_helper.to_jcloud_yaml(temp_file_path, '__4executor/v1.2.3', 'grpc')
+    assert generator_helper.to_jcloud_yaml(temp_file_path, 'executor/v1.2.3', 'grpc')
```

Comment:
Please test the generated values. Please test the newly generated name since the executor name was incorrect.